packages/amplify-python-function-template-provider/resources/crud/event.json

```diff
@@ -0,0 +1 @@
+{ "test": "event" }
```

Comment:
does this event work out of the box with the function? Seems like I remember an event like `{ "path": "/items", "httpMethod": "GET" }` was necessary for it to work

Reply:
You're so right. Thank you for catching that! Updating now.